### PR TITLE
Update and delete

### PR DESCRIPTION
**Pull request by BonkeGcobo**
1) Added more queries

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -5,4 +5,49 @@ SELECT date_of_birth from animals where name = 'Pikachu' or name = 'Agumon';
...
+
+/*How many animals have never tried to escape?*/
+SELECT COUNT(escape_attemps) FROM animals where escape_attemps = 0;
+
+/*What is the average weight of animals?*/
+SELECT AVG(weight_kg) FROM animals; 
+
+/*Who escapes the most, neutered or not neutered animals?*/
+ SELECT SUM(escape_attemps) FROM animals where neutered = TRUE;
+ SELECT SUM(escape_attemps) FROM animals where neutered = FALSE;
```
Comment (AnselemOdims) [inline queries.sql]: - [x] This query will return the sum of the `escape_attempts` for neutered and not neutered in two queries but you are required to write only one query. Please ensure you write one query that will tell who escapes the most. 

Hint
```sql
SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;

```

File: queries.sql [hunk 2]
```diff
@@ -5,4 +5,49 @@ SELECT date_of_birth from animals where name = 'Pikachu' or name = 'Agumon';
...
+
+ /*What is the minimum and maximum weight of each type of animal?*/
+ SELECT MIN(weight_kg) FROM animals;
+ SELECT MAX(weight_kg) FROM animals;
```
Comment (AnselemOdims) [inline queries.sql]: - [x] In this query, you are required to get the minimum and maximum weight of each type of animal in one query. Kindly make sure you use only one query and that they are grouped by type(species)

Hint
```sql
SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
   GROUP BY species;
```

File: queries.sql [hunk 3]
```diff
@@ -5,4 +5,49 @@ SELECT date_of_birth from animals where name = 'Pikachu' or name = 'Agumon';
...
+
+ /*What is the average number of escape attempts per animal type of those born between 1990 and 2000?*/
+ SELECT AVG(escape_attemps) FROM animals where date_of_birth BETWEEN '1990-01-01' and '2000-12-31';
```
Comment (AnselemOdims) [inline queries.sql:53]: - [ ] Please kindly group the query by the animal type(species)

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @BonkeGcobo , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @BonkeGcobo 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 


**Highlights**
- Well done making the required changes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**